.changeset/large-mirrors-teach.md

```diff
@@ -0,0 +1,13 @@
+---
+"@comet/cms-site": minor
```

Comment:
I'm not sure if this is a minor change, since it could possibly break workarounds for the original issue in our projects. For example:

```ts
const Card = styled.div`
    grid-column: 1/2;

    // fix for preview
    > div {
        height: 100%;
    }

    &:nth-child(even) {
        grid-column: 2/2;
    }
`;
```

I believe it would be better to target next for this.

Reply:
Good point 👍🏼